README.md

```diff
@@ -22,42 +22,42 @@
-* TotalCross SDK (or just TotalCross): Is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
+* TotalCross SDK (or just TotalCross): Is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for Android, iOS, WinCE, Windows and desktop, Linux desktop and Linux ARM using Kotlin or Java or any Java Based language.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FTotalCross%2Ftotalcross.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2FTotalCross%2Ftotalcross?ref=badge_shield)
 
 ## How TotalCross SDK works?
 ![Usage flow](https://i.imgur.com/Df3NGui.png)
 
-The developer can use Kotlin or Java to create applications using TotalCross Java API which provides rich GUI components.  The application source code is then compiled resulting in the ByteCode Java, which is converted to our TotalCross Optimized Bytecode, packaged and distributed for the platforms the user specified which can be window and Linux desktop, iOS, Android and Linux arm.
+You can use Kotlin or Java to create applications using TotalCross Java API which provides rich GUI components.  The application source code is then compiled resulting in ByteCode Java, which is converted to TotalCross Optimized Bytecode, packaged and distributed to the platforms the user specifies which can be Window and Linux desktop, iOS, Android and Linux ARM.
 
 ## What is inside this repository?
-O TotalCross SDK is comprised by two main components:
+The TotalCross SDK, comprised by two main components:
 
-* TotalCross Java API -  It is the combination of several apis such as GUI components, database and GPIO to create a single more robust API that guarantees quality and support in application development
-* TCVM. - The heart of totalcross sdk is present our virtual machine, originally idealized in a master's thesis, and already built and improved over 10 years. It's log-based (Java) architecture, bytecode "itself with its own folders" for the most frequent and implemented **almost 100% with C guarantees performance equivalent to native development**. To read more about the TCVM click here.
+* TotalCross Java API - Combination of several APIs such as GUI components, Database and GPIO to create a single more robust API that guarantees quality and support in application development.
+* TCVM -  Totalcross Virtual Machine, originally idealized as a Master's Thesis, has been built and improved over 10 years. It's log-based (Java) architecture, bytecode "itself with its own folders" for the most frequent and implemented **almost 100% with C guarantees performance equivalent to native development**. To read more about the TCVM click here.
```

Comment:
Please add link to "To read more about the TCVM click here" or delete that sentence.

Reply:
added